libphx/src/tex_cube.rs

```diff
@@ -1,25 +1,25 @@
-use crate::internal::Memory::*;
-use crate::Bytes::*;
-use crate::ClipRect::*;
-use crate::Common::*;
-use crate::CubeFace::*;
-use crate::DataFormat::*;
-use crate::Draw::*;
-use crate::GLMatrix::*;
-use crate::Math::Vec3;
-use crate::PixelFormat::*;
-use crate::RenderState::*;
-use crate::RenderTarget::*;
-use crate::Shader::*;
-use crate::ShaderState::*;
-
-use crate::Tex2D_Load::*;
-use crate::Tex2D_Save::*;
-use crate::TexFilter::*;
-use crate::TexFormat::*;
-
-use crate::TimeStamp::*;
-use crate::GL::gl;
+use crate::bytes::*;
```

Comment:
I think this should be called `texcube.rs` for consistency with the other texture types. In shader languages, `texcube` is usually the keyword to denote a cubic texture.

Reply:
Done